shoop/themes/classic_gray/templates/classic_gray/shoop/front/product/detail.jinja

```diff
@@ -106,6 +109,23 @@
                     {% include "shoop/front/product/_detail_attributes_section.jinja" %}
                 </div>
             {% endif %}
+            {% if media_files %}
```

Comment:
They probably shouldn't, because rendering ordered media files vs. public media files on the product detail page may want different layout...

This block here could use a same macro though and if another theme wants to use something else, it can overwrite it. I don't know... IMHO this approach is OK too.